docs/src/getting-started/hello-world.md

```
@@ -0,0 +1,45 @@
+husk build hello.hk
+```
+
+This creates `hello.js`. Run it with Node.js:
+
+```bash
+node hello.js
+```
+
+You should see:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)